rpcs3/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ target_sources(rpcs3
     rpcs3_version.cpp
     stb_image.cpp
     stdafx.cpp
+    rpcn_countries.cpp
```

Comment:
All the other cpp are listed alphabetically...

Reply:
Updated as requested